buildSrc/subprojects/ide/src/main/kotlin/org/gradle/gradlebuild/ide/IdePlugin.kt

```diff
@@ -637,6 +639,38 @@ fun Element.configureCodeStyleSettings() {
 }
 
 
+private
+fun Element.configureInspectionSettings() {
```

Comment:
Please also disable the inspections when using the `idea-ext` plugin by using `settings { inspections { ... } }`. You will need to add the extension method for `inspections`:
```kotlin
fun ProjectSettings.inspections(configuration: Inspection.() -> kotlin.Unit) = (this as ExtensionAware).configure(configuration)
```

Reply:
I added that in the latest commit. However, both before and after my change, my IDEA inspection settings look like this:

<img width="1410" alt="screen shot 2018-09-13 at 16 21 04" src="https://user-images.githubusercontent.com/214207/45494724-fbe3d500-b771-11e8-880b-7d4e3edf99dc.png">

Reply:
Seems like inspections are not yet supported by the `idea-ext` plugin. Can you please check in a
`.idea/inspectionProfiles` directory with this content:
```
Gradle.xml
profiles_settings.xml
```

profiles_settings.xml
```
<component name="InspectionProjectProfileManager">
  <settings>
    <option name="PROJECT_PROFILE" value="Gradle" />
    <version value="1.0" />
  </settings>
</component>
```

`gradle.xml`:
```
<component name="InspectionProjectProfileManager">
  <profile version="1.0">
    <option name="myName" value="Gradle" />
    <inspection_tool class="Convert2Diamond" enabled="false" level="WARNING" enabled_by_default="false" />
    <inspection_tool class="EqualsReplaceableByObjectsCall" enabled="false" level="WEAK WARNING" enabled_by_default="false" />
    <inspection_tool class="SafeVarargsDetector" enabled="false" level="WARNING" enabled_by_default="false" />
    <inspection_tool class="TryFinallyCanBeTryWithResources" enabled="false" level="WARNING" enabled_by_default="false" />
    <inspection_tool class="TryWithIdenticalCatches" enabled="false" level="WARNING" enabled_by_default="false" />
  </profile>
</component>
```